src/app/tabs/settings-tab.js

```diff
@@ -99,166 +150,183 @@ function SettingsTab (appAPI = {}, appEvents = {}, opts = {}) {
-      return
+    function onchangeLoadVersion (event) {
+      self.data.selectedVersion = self._view.versionSelector.value
+      self._updateVersionSelector()
     }
-    url = 'https://solc-bin.ethereum.org/bin/' + version
+    function onchangePersonal (event) {
+      self._api.config.set('settings/personal-mode', !self._api.config.get('settings/personal-mode'))
+    }
+    return self._view.el
   }
-  var isFirefox = typeof InstallTrigger !== 'undefined'
-  if (document.location.protocol !== 'file:' && Worker !== undefined && isFirefox) {
-    // Workers cannot load js on "file:"-URLs and we get a
-    // "Uncaught RangeError: Maximum call stack size exceeded" error on Chromium,
-    // resort to non-worker version in that case.
-    compiler.loadVersion(true, url)
-    setVersionText('(loading using worker)', el)
-  } else {
-    compiler.loadVersion(false, url)
-    setVersionText('(loading)', el)
+  setVersionText (text) {
+    const self = this
+    self.data.version = text
+    if (self._view.version) self._view.version.innerText = text
+  }
+  _updateVersionSelector () {
+    const self = this
+    self._view.versionSelector.innerHTML = ''
+    self._view.versionSelector.appendChild(yo`<option disabled selected>Select new compiler version</option>`)
+    self.data.allversions.forEach(build => self._view.versionSelector.appendChild(yo`<option value=${build.path}>${build.longVersion}</option>`))
+    self._view.versionSelector.removeAttribute('disabled')
+    self._components.queryParams.update({ version: self.data.selectedVersion })
+    var url
+    if (self.data.selectedVersion === 'builtin') {
+      var location = window.document.location
+      location = location.protocol + '//' + location.host + '/' + location.pathname
+      if (location.endsWith('index.html')) location = location.substring(0, location.length - 10)
+      if (!location.endsWith('/')) location += '/'
+      url = location + 'soljson.js'
+    } else {
+      if (self.data.selectedVersion.indexOf('soljson') !== 0 || helper.checkSpecialChars(self.data.selectedVersion)) {
+        return console.log('loading ' + self.data.selectedVersion + ' not allowed')
+      }
+      url = 'https://ethereum.github.io/solc-bin/bin/' + self.data.selectedVersion
+    }
+    var isFirefox = typeof InstallTrigger !== 'undefined'
+    if (document.location.protocol !== 'file:' && Worker !== undefined && isFirefox) {
+      // Workers cannot load js on "file:"-URLs and we get a
+      // "Uncaught RangeError: Maximum call stack size exceeded" error on Chromium,
+      // resort to non-worker version in that case.
+      self._opts.compiler.loadVersion(true, url)
+      self.setVersionText('(loading using worker)')
+    } else {
+      self._opts.compiler.loadVersion(false, url)
+      self.setVersionText('(loading)')
+    }
+  }
+  fetchAllVersion (callback) {
+    var self = this
+    minixhr('https://ethereum.github.io/solc-bin/bin/list.json', function (json, event) {
```

Comment:
the url changed. that should be `https://solc-bin.ethereum.org/bin/list.json` 
https://github.com/ethereum/remix-ide/pull/1237/files#diff-c03f1d552b7bd6b97cb0d9bd89a82e3eL190